projects/ids-enterprise-ng/package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@types/jquery": "3.3.24",
     "@types/d3": "4.13.0",
-    "ids-enterprise": "4.17.0-beta.0",
+    "ids-enterprise": "4.18.0-dev.20190319",
```

Comment:
I thought we wanted to stay on 4.17 for 5.2.0 ?

Reply:
nice catch, puting that back